#
#
# patch "libpurple/protocols/zephyr/zephyr.c"
#  from [dc185e6df4e010dbf800523743ba226ad40be76e]
#    to [b282d503470d98f20ea7008e1742ee7721947bd7]
#
============================================================
--- pidgin-2.4.0/libpurple/protocols/zephyr/zephyr.c	dc185e6df4e010dbf800523743ba226ad40be76e
+++ pidgin-2.4.0.new/libpurple/protocols/zephyr/zephyr.c	b282d503470d98f20ea7008e1742ee7721947bd7
@@ -2235,6 +2235,9 @@ static const char *zephyr_normalize(cons
 	char *tmp;
 
 	gc = purple_account_get_connection(account);
+	if (gc == NULL)
+		return NULL;
+
 	tmp = local_zephyr_normalize(gc->proto_data, who);
 
 	if (strlen(tmp) >= sizeof(buf)) {
